Core/include/Acts/Geometry/DetectorVolume.hpp

```diff
@@ -40,6 +41,10 @@ using PortalGenerator = Delegate<std::vector<std::shared_ptr<Portal>>(
     const Transform3&, const VolumeBounds&,
     const std::shared_ptr<DetectorVolume>&)>;
 
+using BoundingBox =
```

Comment:
Should this typedef be scoped to the `DetectorVolume` class?

Reply:
Yep! Done.